public/content/roadmap/pectra/index.md

```diff
@@ -1,12 +1,14 @@
 ---
-title: Pectra
-description: Learn about Pectra - the next Ethereum protocol upgrade
+title: Prague-Electra (Pectra)
+description: Learn about the Pectra protocol upgrade
 lang: en
 ---
 
 # Pectra {#pectra}
 
-Pectra is an upcoming Ethereum protocol upgrade that brings new functionality and changes to the Ethereum network. Following [Dencun](/roadmap/dencun/), this is another major upgrade to both the execution and consensus layer of Ethereum. The shortened name Pectra is a combination of Prague and Electra, which are the respective names for the execution and consensus layer specification changes. Together, these changes bring a number of improvements to Ethereum users, developers and validators.
+Prague-Electra (Pectra) is an upgrade to the Ethereum network, following [Dencun](/roadmap/dencun/). This major upgrade brings changes to both the execution and consensus layer of Ethereum. The shortened name Pectra is a combination of Prague and Electra, which are the respective names for the execution and consensus layer specification changes. Together, these changes bring a number of improvements to Ethereum users, developers and validators.
```

Comment:
Would make this whole first sentence past-tense now.. and maybe simplify a touch:

```suggestion
The Pectra network upgrade followed [Dencun](/roadmap/dencun/) and brought changes to both the execution and consensus layer of Ethereum. The shortened name Pectra is a combination of Prague and Electra, which are the respective names for the execution and consensus layer specification changes. Together, these changes bring a number of improvements to Ethereum users, developers and validators.
```

(We explain the naming breakdown in a dedicated sentence, so not sure it's needed at the start imo)

Reply:
Agree! Thanks :)